totalRP3/Resources/InterfaceIcons.lua

```diff
@@ -84,6 +84,8 @@ TRP3_InterfaceIcons = {
 	BloodElfMale             = { "achievement_character_bloodelf_male", "inv_weapon_bow_38", DEFAULT_ICON_NAME },
 	DarkIronDwarfFemale      = { "ability_racial_foregedinflames", DEFAULT_ICON_NAME },
 	DarkIronDwarfMale        = { "ability_racial_fireblood", DEFAULT_ICON_NAME },
+	DracthyrFemale           = { "inv_dracthyrhead01", "inv_dracthyrhead11"},
```

Comment:
```suggestion
	DracthyrFemale           = { "ui_dracthyr", DEFAULT_ICON_NAME},
```

I would suggest using the default Dracthyr icon `ui_dracthyr` as I feel like it more closely matches what I'd expect the default icon to be.

Also redefine the second entry as `DEFAULT_ICON_NAME` because (I think) it's used as a fallback if the first icon can't be found. I might be wrong about this though.